src/constants/Message.js

```diff
@@ -0,0 +1,15 @@
+export const CALCULATER = Object.freeze({
+  input_prompt: '덧셈할 문자열을 입력해 주세요.\n',
+  result: calculatedResult => `결과 : ${calculatedResult}`,
```

Comment:
```suggestion
  result: calculatedResult => `결과 : ${calculatedResult}`, // 유틸일까? 상수일까? 🤔
```
저도 예전에 이렇게 동적으로 값을 받아서 사용했던 경험이 있었는데, 상수라는 개념이 변하지 않는 값을 모아뒀는데 이렇게 동적으로 받아서 사용하면 사실상 유틸쪽이 가깝지 않나 고민이 되었고 최종적으로 유틸쪽에 더 가깝다고 판단하게 되었는데 이 부분에 대해서 지섭님의 의견이 궁금합니다!!

Reply:
출력 문장을 한번에 관리할려는 생각에 이렇게 사용해봤는데 유틸로 생각될지는 몰랐습니다..! 
다만 출력문구는 그대로고 값만 할당되는 것이라 문제가 없다고 생각했습니다. 
제가 생각하는 const는 재할당과 재선언을 할 수 없는 상수. 이기에 이렇게 사용했던 것같아요.
하지만 우혁님의 리뷰를 보고나니 해당 코드는 유틸리티 함수가 맞을 것같아요. 상수로 사용할 것이였으면 분리해서 사용하는 것이 맞는 것 같습니다. 좋은 의견 감사해요!